demisto_sdk/commands/create_artifacts/content_creator.py

```diff
@@ -376,28 +419,31 @@ def copy_test_files(self, test_playbooks_dir=TEST_PLAYBOOKS_DIR):
             if os.path.isdir(path):
                 non_circle_tests = glob.glob(os.path.join(path, '*'))
                 for new_path in non_circle_tests:
-                    print(f'copying path {new_path}')
-                    new_file_path = os.path.join(self.test_bundle, os.path.basename(new_path))
-                    shutil.copyfile(new_path, new_file_path)
-                    self.add_from_version_to_yml(new_file_path)
+                    if os.path.isfile(new_path) and self.check_from_version(new_path):
```

Comment:
when will glob return a file that doesn't exist?
or is that for skipping directories?

Reply:
skipping directories from what I gather - I did not change that part